components/dashboard/src/dedicated-setup/OrgNamingStep.tsx

```diff
@@ -28,7 +28,16 @@ export const OrgNamingStep: FC<Props> = ({ onComplete }) => {
         if (org.data) {
             updateOrg.mutate({ name: orgName }, { onSuccess: onComplete });
         } else {
-            createOrg.mutate({ name: orgName }, { onSuccess: onComplete });
+            createOrg.mutate(
+                { name: orgName },
+                {
+                    onSuccess: (newOrg) => {
+                        // Need to manually set active-org here so it's returned via subsequent useCurrentOrg() calls
+                        localStorage.setItem("active-org", newOrg.id);
```

Comment:
In other places we do a redirect, in order to achieve this, which seems to be rather destructive method. Would it make sense to expose a `useChangeCurrentOrg` hook from the org-query module?

Reply:
what triggers the loading of the changed value?